nntrainer/layers/layer_node.h

```diff
@@ -661,11 +661,47 @@ class LayerNode final : public ml::train::Layer, public GraphNode {
    */
   std::unique_ptr<LayerNode> cloneConfiguration();
 
+  /**
+   * @brief Set if the layer needs to do derivative calculation
+   *
+   * @param nb true if the layer needs to do backwarding, else false
+   */
+  void needsCalcDerivative(bool nb) {
```

Comment:
so.. we have..
1. supportBackwarding
2. needsBackwarding --> needsCalcDerivative & needsCalCGradient?

Reply:
supportBackwarding is acutally supportCalcDerivative (didn't change this as its easier to understand for the layer developers), but internally, needsBackwarding is created and split into two.

Reply:
Thanks for the explanation.